request_raw.go

```
@@ -0,0 +1,631 @@
+			}
+
+			// 如果字段是结构体类型，添加到队列
+			if field.Kind() == reflect.Struct {
+				queue = append(queue, field)
+			}
+		}
+	}
+
+	return nil
+}
+func formatRequestBody(body interface{}) interface{} {
```

golangci-lint (1.64.8)

func `formatRequestBody` is unused

(unused)